Fst2List.cpp

```diff
@@ -772,6 +773,23 @@ class CFstApp {
     inputPtrCnt = outputPtrCnt = inBufferCnt = outBufferCnt = 0;
   }
 
+  /**
+    * print one space when the morphological mode is off
+  **/
+  void outOneSpace() {
```

Comment:
What about https://github.com/UnitexGramLab/unitex-core/pull/82/files#r297094591

Reply:
I think I misunderstood your comment. What changes should I make?

Reply:
What about `appendSingleSpace` instead of `outOneSpace` ?

Reply:
It seems better than outOneSpace